### PR TITLE
Transparent slime blood

### PR DESCRIPTION
**Pull request by kugamo**
## What Does This PR Do

Slime bloods are strangely opaque to come from a translucent being, this PR makes their blood splatters translucent.

## Why It's Good For The Game

visual consistency. opaque blood coming from translucent beings make no sense

## Images of changes

transparentbloods

## Changelog
:cl:
tweak: Slimeperson blood now translucent, like their bodies are
/:cl:

### CODE REVIEW

File: code/game/objects/effects/decals/Cleanable/aliens.dm [hunk 1]
```diff
@@ -43,6 +43,7 @@
 	desc = "It's a transparent semi-liquid from a slime or slime person. Don't lick it."
 	basecolor = "#0b8f70"
 	bloodiness = MAX_SHOE_BLOODINESS
+	alpha = 150
```
Comment (hal9000PR) [inline code/game/objects/effects/decals/Cleanable/aliens.dm:46]: make this into a define please. something like `BLOOD_SLIME_ALPHA`

File: code/game/objects/effects/decals/Cleanable/tracks.dm [hunk 2]
```diff
@@ -118,13 +121,15 @@ GLOBAL_LIST_EMPTY(fluidtrack_cache)
...
 		FP.basecolor = H.feet_blood_color
+		FP.base_alpha = H.bloody_feet["blood_alpha"]
 		if(H.blood_DNA)
 			FP.blood_DNA = H.blood_DNA.Copy()
 	else if(istype(A, /obj/item/clothing/shoes))
 		var/obj/item/clothing/shoes/S = A
 		FP.blood_state = S.blood_state
 		FP.bloodiness = S.bloody_shoes[S.blood_state] - BLOOD_LOSS_IN_SPREAD
 		FP.basecolor = S.blood_color
+		FP.base_alpha = S.bloody_shoes["blood_alpha"]
```
Comment (hal9000PR) [inline code/game/objects/effects/decals/Cleanable/tracks.dm:132]:
```suggestion
		FP.base_alpha = S.bloody_shoes[BLOOD_STATE_BASE_ALPHA]
```
use your defines. do this for all instances of this please.

File: code/game/objects/effects/decals/cleanable.dm [hunk 3]
```diff
@@ -28,6 +28,7 @@
 				add_blood = bloodiness
 			bloodiness -= add_blood
 			S.bloody_shoes[blood_state] = min(MAX_SHOE_BLOODINESS, S.bloody_shoes[blood_state] + add_blood)
+			S.bloody_shoes["blood_alpha"] = 150 * (alpha/255)
```
Comment (hal9000PR) [inline code/game/objects/effects/decals/cleanable.dm:31]: I am not quite sure what is going on here, why do you multiply it by 150? won't this apply to all blood and not just slime jelly?

File: code/modules/clothing/clothing.dm [hunk 4]
```diff
@@ -454,7 +454,7 @@ BLIND     // can't see anything
 	slot_flags = SLOT_FEET
 
 	var/blood_state = BLOOD_STATE_NOT_BLOODY
-	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0, BLOOD_STATE_XENO = 0, BLOOD_STATE_NOT_BLOODY = 0)
+	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0, BLOOD_STATE_XENO = 0, BLOOD_STATE_NOT_BLOODY = 0, BLOOD_STATE_BASE_ALPHA = 150)
```
Comment (hal9000PR) [inline code/modules/clothing/clothing.dm:457]: again, shouldn't this default to 255? else it will make all blood translucent?

**Review by hal9000PR — CHANGES_REQUESTED** [review]

File: code/modules/clothing/clothing.dm (hunk 4 above)
Comment (kugamo) [reply code/modules/clothing/clothing.dm:457]: no the variable should default to BLOODY_FOOTPRINT_BASE_ALPHA this is what Im using it to alter

File: code/game/objects/effects/decals/cleanable.dm (hunk 3 above)
Comment (kugamo) [reply code/game/objects/effects/decals/cleanable.dm:31]: default is 150, I multiply it by a fraction to get the tracks to be more transparent. Maybe I should just use BLOODY_FOOTPRINT_BASE_ALPHA here

File: code/game/objects/effects/decals/Cleanable/tracks.dm (hunk 2 above)
Comment (kugamo) [reply code/game/objects/effects/decals/Cleanable/tracks.dm:132]: haha ok =

**Review by AffectedArc07 — APPROVED** [review]

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**